integration/bun/index.ts

```diff
@@ -0,0 +1,16 @@
+import Replicate from "replicate";
+
+const replicate = new Replicate({
+  auth: process.env.REPLICATE_API_TOKEN,
+});
+
+export default async function main() {
+  return await replicate.run(
+    "replicate/hello-world:5c7d5dc6dd8bf75c1acaa8565735e7986bc5b66206b55cca93cb72c9bf15ccaa",
+    {
+      input: {
+        text: "Brünnhilde Bun",
+      },
+    }
+  );
```

Comment:
That's a great idea. I did find an issue in Bun's (lack of) support for `TextDecoderStream` (https://github.com/oven-sh/bun/issues/5648), but I'd like to take that on in a follow-up, since this is in a good state right now as-is.